src/dmd/vsoptions.d

```diff
@@ -635,18 +722,46 @@ const(char)* detectVSInstallDirViaCOM()
         return null;
     scope(exit) instances.Release();
 
+    static struct WrappedBString
```

Comment:
Nit: ok, but seems a bit excessive for dealing with releasing two strings. scope(exit) and a swap function might do the trick, too.

Reply:
Yeah it might be excessive, but after having to explain the previous variant to you, I went with this.